.github/workflows/main.yml

```diff
@@ -0,0 +1,53 @@
+name: sphinx-auto-update
+on:
+  schedule:
+    - cron:  '0 0 * * *'
+jobs:
+  script:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Get Job URL
+      uses: Tiryoh/gha-jobid-action@v0
+      id: jobs
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        job_name: script
+    - name: checkout with submodule
+      uses: actions/checkout@v2
+      with:
+        submodules: true
+        token: ${{ secrets.PERSONAL_ACCESSTOKEN }}
+    - name: Setup SSH
+      uses: MrSquaare/ssh-setup-action@v1
+      with:
+        host: github.com
+        private-key: ${{ secrets.SSH_PRIVATE_KEY }}
+    - name: install
+      run: |
+        (cd sphinx; git fetch origin; git checkout master; git reset --hard origin/master; git branch -a)
+        pip3 install -U pip setuptools
+        pip3 install -r ./requirements.txt
+    - name: script
+      env:
+        TX_TOKEN: ${{ secrets.TX_TOKEN }}
+        SPHINXINTL_TRANSIFEX_USERNAME: sphinxjp
+        SPHINXINTL_TRANSIFEX_PROJECT_NAME: sphinx-doc
+      run: |
+        sh ./locale/update.sh
+    - name: git_config
+      run: |
+        git branch -a
+        git checkout master
```

Comment:
I don't consider these two lines above is not a part of "git_config". How about move them into "commit" step?